src/Uno.UWP/Storage/ApplicationDataContainer.wasm.cs

```diff
@@ -20,165 +24,87 @@ public partial class ApplicationDataContainer
 
 		private class FilePropertySet : IPropertySet
 		{
-			private const string UWPFileName = ".UWPAppSettings";
-			private readonly Dictionary<string, string> _values = new Dictionary<string, string>();
-			private readonly string _folderPath;
-			private readonly string _filePath;
+			private readonly ApplicationDataLocality _locality;
 
 			public FilePropertySet(ApplicationData owner, ApplicationDataLocality locality)
 			{
-				StorageFolder folder;
-				switch (locality)
-				{
-					case ApplicationDataLocality.Local:
-						folder = owner.LocalFolder;
-						break;
-
-					case ApplicationDataLocality.Roaming:
-						folder = owner.RoamingFolder;
-						break;
-					case ApplicationDataLocality.LocalCache:
-						folder = owner.LocalCacheFolder;
-						break;
-
-					case ApplicationDataLocality.Temporary:
-						folder = owner.TemporaryFolder;
-						break;
-
-					default:
-						throw new ArgumentOutOfRangeException(nameof(locality));
-				}
-
-				_folderPath = folder.Path;
-				_filePath = Path.Combine(folder.Path, UWPFileName);
-
-				ReadFromFile();
+				_locality = locality;
 			}
 
-			public object this[string key]
+			public object? this[string key]
 			{
 				get
 				{
-					if(_values.TryGetValue(key, out var value))
+					if (ApplicationDataContainerInterop.TryGetValue(_locality, key, out var value))
 					{
+						Console.WriteLine($"key={key} value={value}");
 						return DataTypeSerializer.Deserialize(value);
 					}
 
+					Console.WriteLine($"key={key} value=**null**");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5609373)